h2o-core/src/main/java/water/H2O.java

```diff
@@ -1775,11 +1788,12 @@ public static void main( String[] args ) {
     Log.info("User name: '" + H2O.ARGS.user_name + "'");
 
     // Register with GA or not
-    List<String> gaidList = JarHash.getResourcesList("gaid");
+    long time3 = System.currentTimeMillis();
+    List<String> gaidList;  // fetching this list takes ~100ms
     if((new File(".h2o_no_collect")).exists()
             || (new File(System.getProperty("user.home")+File.separator+".h2o_no_collect")).exists()
             || ARGS.ga_opt_out
-            || gaidList.contains("CRAN")
+            || (gaidList = JarHash.getResourcesList("gaid")).contains("CRAN")
```

Comment:
Maybe; the idea was that fetching the resource list takes non-trivial amount of time, so should be delayed until it is really necessary. A cleaner solution would be to remove this test altogether, and rely on some other mechanism to provide GA exception for CRAN